src/components/Book.js

```diff
@@ -25,6 +37,8 @@ const Book = ({ title, author }) => {
 Book.propTypes = {
   title: PropTypes.string.isRequired,
   author: PropTypes.string.isRequired,
+  bookId: PropTypes.string.isRequired,
```

Comment:
- [ ] This line is showing an error on the console because bookId is not a string. Please set PropTypes as a number to remove this error.
example: `bookId: PropTypes.number.isRequired,`

### See the image of the error.


![image](https://user-images.githubusercontent.com/83803414/162434697-aa857afc-5b6a-462f-9bd8-5490aa5db597.png)

Reply:
fixed! 😄